File: src/main/intellibitz/sted/event/FontMapEntriesChangeEvent.java
package intellibitz.sted.event;

import intellibitz.sted.fontmap.FontMapEntries;

import javax.swing.event.ChangeEvent;

public class FontMapEntriesChangeEvent
        extends ChangeEvent {
    public FontMapEntriesChangeEvent(FontMapEntries fontMapEntries) {
        super(fontMapEntries);
    }

    public FontMapEntries getFontMapEntries() {
        return (FontMapEntries) getSource();
    }

}

File: src/main/intellibitz/sted/event/FontMapEntriesChangeEvent.kt
package sted.event

import sted.fontmap.FontMapEntries
import javax.swing.event.ChangeEvent

class FontMapEntriesChangeEvent(fontMapEntries: FontMapEntries) : ChangeEvent(fontMapEntries) {
    val fontMapEntries: FontMapEntries
        get() = getSource() as FontMapEntries
}